feature/_version.py

```diff
@@ -2,4 +2,4 @@
 # Copyright FMR LLC <opensource@fidelity.com>
 # SPDX-License-Identifier: Apache-2.0
 
-__version__ = "1.1.2"
\ No newline at end of file
+__version__ = "1.2.0"
```

Comment:
Cool. I am a maintainer of selective in pypi. I can upload the new version to pypi @skadio . And I see we should also create a new release on github.